### PR TITLE
fix: show warning and skip Send event creation if `DISABLE_SAVE_AND_RETURN` feature flag is enabled

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Addresses this error: https://opensystemslab.slack.com/archives/C01E3AC0C03/p1699868054247069

Testing:
- With no feature flags toggled, a published flow will create "Send" events as expected. There is no warning banner.
- With the `DISABLE_SAVE_AND_RETURN` feature flag toggled, you can still proceed through Send to the Confirmation page, but no events have been created in Hasura. The warning banner is also visible.

### CODE REVIEW

File: scripts/seed-database/write/flows.sql [hunk 1]
```diff
@@ -46,7 +46,7 @@ SET
   version = EXCLUDED.version,
   settings = EXCLUDED.settings,
   copied_from = EXCLUDED.copied_from,
-  analytics_link = null;
+  analytics_link = NULL;
```
Comment (jessicamcinchak) [inline scripts/seed-database/write/flows.sql:49]: Sorry for bundling, but this is a followup syntax fix to #2414

File: editor.planx.uk/src/components/FeatureFlagBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+              Hide
+            </Button>
+          </Container>
+        </TestEnvironmentWarning>
+      )}
+    </>
+  );
+};
+
+export default FeatureFlagBanner;
```
Comment (jessicamcinchak) [inline editor.planx.uk/src/components/FeatureFlagBanner.tsx:54]: This displays like this when condition is met:
Screenshot from 2023-11-13 15-40-01

Open to language suggestions here!

File: scripts/seed-database/write/flows.sql (hunk 1 above)
Comment (jessicamcinchak) [reply scripts/seed-database/write/flows.sql:49]: Confirmed this works here https://github.com/theopensystemslab/planx-new/actions/runs/6852179815

**Review by DafyddLlyr — APPROVED** [review]
Perfect! 👌

**Pull request merged**